index.md

```diff
@@ -1444,6 +1444,30 @@ Numeric Replies:
     <p>NOTE: The <tt>KILL</tt> message is weird, and I need to look at it more closely, add some examples, etc.</p>
 </div>
 
+### SQUIT message
+
+         Command: SQUIT
+      Parameters: <server> <comment>
+
+The SQUIT command is available only to operators.  It is used to disconnect server links.
+
+The `<comment>` SHOULD be supplied by all operators who execute a SQUIT for a remote server.
+
+The server ordered to disconnect its peer generates a [`WALLOPS` message ](#wallops-message) with `<comment>` included, so that other users may be aware of the reason of this action.
```

Comment:
Is this relevant anymore? I don't think we do this at the least.

Reply:
Indeed, thx